packages/runtime/react/index.tsx

```diff
@@ -0,0 +1,25 @@
+import ReactDOM from 'react-dom';
+import * as React from 'react';
+
+export const renderTemplate = (template: any, model: any, component?: any): void => {
+  template(model, model.container);
+};
+export const hasTemplate = (name: any, props: any, component?: any): boolean => !!props[name];
+export const getWrapperTemplate = (TemplateProp: any) => (data: any) => {
```

Comment:
index for re-export. I suggest moving every function to the separated module file

Reply:
done